File: readability/readability.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //getting user input
    string text = get_string("Text: ");

    //count the number of letters
    int num_letters = 0;
    int i = 0;

    while(text[i] != '\0')
    {
        if( ((text[i] >= 65) && (text[i] <= 90)) ||  ((text[i] >= 97) && (text[i] <= 122)))
        {
            num_letters++;
        }
        i++;
    }

    printf("Number of letters: %d", num_letters);

}

File: problem-sets/sentimental-readability/readability.py
# TODO

from cs50 import get_string

# getting user input

text = get_string("Text: ")

# count the number of letters
num_letters = len(text)
# print(num_letters)

# count the number of words
num_words = len(text.split())
# print(num_words)

# count the number of sentences
num_sentences = len(text.split('.'))
print(num_sentences)